packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -0,0 +1,116 @@
+.plan-details-component {
+  background: #fff;
+  border-radius: 8px;
+  box-shadow: 0 12px 18px 2px
+rgba(34, 0, 51, 0.04), 0 6px 22px 4px rgba(7, 48, 114, 0.12), 0 6px 10px -4px
+rgba(14, 13, 26, 0.12);
+  font-size: 15px;
+  margin: 0 auto;
+  padding: 20px 20px 0;
+  width: 342px;
+
+  .plan-details-header-wrap {
+    display: flex;
+  }
+
+  .plan-details-heading-wrap {
+    display: flex;
+    flex-direction: column;
+    padding-left: 10px;
+    text-align: left;
+
+    .plan-details-product {
+      /* Grey/Grey 50 */
+      color: rgba(12, 12, 13, 0.8);
+      font-weight: 600;
+      line-height: 19px;
+      word-wrap: break-word;
+    }
+
+    .plan-details-description {
+      color: rgba(12, 12, 13, 0.6);
+      margin-top: 4px;
+    }
+  }
+
+  header {
+    border-bottom: 1px solid#e7e7e7;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0;
+    padding-bottom: 20px;
+  }
+
+  header p {
+    margin: 0;
+  }
+
+  header img {
+    display: block;
+    height: 64px;
+    width: 64px;
+  }
+
+  .plan-details-list {
+    border-bottom: 1px solid#e7e7e7;
```

Comment:
Missing a space here as well as line 37.

Can we set some of these colors to variables?

Reply:
We don't seem to be using variables anywhere on payment server. We definitely should be though. I feel like Danny had a bug for this in the maintenance board

Reply:
👍 We're importing the variables from the content server in at least a couple places, but, as I've said before I'm going to revamp our SCSS with the settings redesign and reviewing/setting variables is part of that. I'd already planned on going through payments as part of that to standardize us across the board so I'd say let's not spend too much time on that but import them where it makes sense.